recipes/eip712/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "eip712" %}
+{% set version = "0.2.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/ApeWorX/{{ name }}/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: e388e6281048bda1a575a2ca75e6f721fa771c67979554b6937758eae52335cf
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vvv
+  number: 0
+
+requirements:
+  host:
+    - python >=3.8.0
+    - hexbytes
```

Comment:
Is this required in host as well?

Reply:
No, it is not and is removed in the latest push